
 Sun, 30 Jul 2023 10:51:11 GMT

COMMAND LINE:
 /usr/bin/python3 fibonacci.python3-3.python3 1000000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "/home/dundee/work/pybenchmarks/bencher/tmp/fibonacci/tmp/fibonacci.python3-3.python3", line 22, in <module>
    print(fib(int(sys.argv[1])))
ValueError: Exceeds the limit (4300 digits) for integer string conversion; use sys.set_int_max_str_digits() to increase the limit
